package.json

```
@@ -5,10 +5,11 @@
   "main": "index.js",
   "exports": {
     ".": {
-      "import": "./index.js",
-      "types": "./types/index.d.ts"
+      "types": "./types/index.d.ts",
+      "default": "./index.js"
```

Since the package is ESM-first, I would suggest leaving all we already have as-is, but adding `require: "./index.js"` (after the `import` property) to support CJS, and not using the general fallback. Will this fix the issue with your app?

```suggestion
      "import": "./index.js",
      "require": "./index.js",
      "types": "./types/index.d.ts"
```

Changing `import` to `default` is not strictly wrong, but `import` is the more standard and explicit way to specify the ESM entry point.

---

That works too, thanks!

---

Yay!